src/arch/arm/armmlib/exception_handlers.c

```diff
@@ -41,6 +58,7 @@ void exc_default_handler(struct exc_saved_base_ctx *ctx, int xpsr) {
 		hard_fault();
 		break;
 	default:
-		break;
+		print_fault_status();
```

Comment:
It looks almost the same as `hard_fault()`, may be it can be used here?

Reply:
Yes, thank you, I simplified the logic here.